<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticTextDlg</class>
 <widget class="QWidget" name="StaticTextDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>文本属性</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>142</y>
      <width>57</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>大 小：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FontSize">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>54</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字 体：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>字 距：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="WordInterval">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="font">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_Alignment">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>居中</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>左对齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>右对齐</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>84</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>对 齐：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FontColor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>172</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>颜 色：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="adaptText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>自适应</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>语言：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_languagecmb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>填充</string>
   </property>
   <widget class="QComboBox" name="Pattern">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>样    式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>前 景 色：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FrontColor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>64</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>透 明 度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Alpha">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="BackColor">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>背 景 色：</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="m_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>所有语言都使用第一种语言的文本</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>281</width>
     <height>351</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_underlinecheck">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IfBold">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IfItalic">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
